components/inputs/input-select-styles.js

```diff
@@ -25,7 +25,7 @@ export const selectStyles = css`
 		max-height: calc(2rem + 2px);
 		vertical-align: middle;
 	}
-	:host([dir='rtl']) .d2l-input-select {
+	:host([dir="rtl"]) .d2l-input-select {
```

Comment:
Are the quotes something we can no longer control? Are single quotes not technically allowed?

Reply:
I guess double quotes are more correct from an HTML perspective.

Reply:
Yeah, I went with the double quotes for that reason. I think even in CSS it might technically more correct. I was bothered that it applies to things like `content` too, but I don't have a super strong opinion on any of them really.